iprange/sortip.go

```diff
@@ -0,0 +1,36 @@
+package iprange
+
+import "net"
+
+// Asc implements sorting in ascending order for IP addresses
+type asc []net.IP
+
+func (a asc) Len() int {
+	return len(a)
+}
+
+func (a asc) Swap(i, j int) {
+	a[i], a[j] = a[j], a[i]
+}
+
+func (a asc) Less(i, j int) bool {
```

Comment:
This assumes ipv4, right? It would be safer to ensure this is an IPv4 address.

Also, I think we could convert this IP into an integer and compare both values instead of comparing octects.

```
fmt.Println("ip2int: ", binary.BigEndian.Uint32([]byte{127, 0, 0, 1}))
```

Code would be easier to read, I think.